docs/docs/developer-guides/cosmwasm-cli.md

```diff
@@ -49,25 +81,32 @@ ccw new --name get-started --description "Get started with CosmWasm"
 ```
 
 See [here](./cosmwasm/new-project.md) for more.
+-->
 
-## Upload a CosmWasm contract
 
-For interacting with `pallet-cosmwasm`, `substrate` subcommand is used. To be able
-to call your contracts, you need to upload them to the chain first. There are several
-sources to upload your contracts:
+### Upload a CosmWasm contract
 
-### 1. Upload a local contract binary
+For interacting with `pallet-cosmwasm`, the `substrate` subcommand is used. To be able
+to call your contracts, you need to upload them to the chain first. The difference between running on a local devnet and on Picasso Rococo is to replace '-n Alice' with your seed phrase in the commands and the RPC endpoints, an example is provided below during the upload of a local contract binary.
+
+There are several sources to upload your contracts:
+
+#### 1. Upload a local contract binary
 
 You need to specify the file path and the signer to be able to upload a contract
 from the file path. Extrinsics must be called by a signed entity in `pallet-cosmwasm`.
-For now, the examples will use development accounts for signing extrinsics, but
-we will explain it further later.
 
 ```sh
-ccw -n alice upload -f /path/to/file.wasm
+# On Picasso local
+cargo run substrate -c ws://127.0.0.1:9988 -n alice tx upload --file_path ./cw20_base.wasm
```

Comment:
substitute file_path to file-path everywhere

Reply:
oh my bad, edited